thirdparty/google_crashpad_client/CMakeLists.txt

```diff
@@ -244,6 +244,11 @@ add_library(gcrashpad STATIC
     ${CRASHPAD_SRC}
 )
 
+if (NOT MSVC)
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-unused-parameter -Wno-multichar -Wno-restrict -Wno-attributes")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unused-parameter -Wno-multichar -Wno-restrict -Wno-attributes")
+endif (NOT MSVC)
+
```

Comment:
This will clash with #7010

Reply:
Add:
```
else (NOTE MSVC)
    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4100 /wd4127 /wd4201 /wd4996")
```

Reply:
Okay!

Reply:
That would still cause a merge conflict, but at one that is obvious and easy to fix

Reply:
done

Reply:
Looks like you could remove your change to `thirdparty/google_crashpad_client/CMakeLists.txt ` from your PR since mine does what both of ours needs to do now.

Reply:
Will do, once I touch master again ;-)